.gitignore

```diff
@@ -1,3 +1,5 @@
+## JS
+
```

Comment:
?

Reply:
I combined the root gitignores and the backend gitignores, so I separated them out

Reply:
perhaps JS wasn't the best section name to give that